src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+
+.card {
+  box-sizing: border-box;
```

Comment:
move it to the `*` selector

Reply:
![image](https://user-images.githubusercontent.com/99331793/197971055-a3d035c0-fb3d-455b-a81b-cad247006fda.png)
fix it